pkg/controller/kfservice/reconcilers/istio/virtualservice_reconciler.go

```diff
@@ -14,107 +14,101 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package knative
+package istio
 
 import (
 	"context"
 	"fmt"
-	"github.com/kubeflow/kfserving/pkg/constants"
 
 	"github.com/kubeflow/kfserving/pkg/apis/serving/v1alpha2"
-	"github.com/kubeflow/kfserving/pkg/controller/kfservice/resources/knative"
+	"github.com/kubeflow/kfserving/pkg/controller/kfservice/resources/istio"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/equality"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
+	istiov1alpha3 "knative.dev/pkg/apis/istio/v1alpha3"
 	"knative.dev/pkg/kmp"
-	knservingv1alpha1 "knative.dev/serving/pkg/apis/serving/v1alpha1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
 )
 
-type RouteReconciler struct {
-	client client.Client
-	scheme *runtime.Scheme
+var log = logf.Log.WithName("VirtualServiceReconciler")
```

Comment:
klog!

Reply:
i thought controller runtime logger is better(https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md), klog seems like rebrand of glog.

Reply:
can we leave consolidating to one for a future date? this PR is already quite long in the tooth...